.gitpod.yml

```diff
@@ -11,14 +11,16 @@ tasks:
       cmake -S extern/test_bmi_c -B extern/test_bmi_c/cmake_build
       cmake -S extern/test_bmi_cpp -B extern/test_bmi_cpp/cmake_build
       cmake -S extern/test_bmi_fortran/ -B extern/test_bmi_fortran/cmake_build
-      cmake -S extern/evapotranspiration/ -B extern/evapotranspiration/cmake_build
+      cmake -S extern/sloth -B extern/sloth/cmake_build
+      cmake -S extern/evapotranspiration/evapotranspiration -B extern/evapotranspiration/cmake_build
```

Comment:
Not going to worry about this being out of scope (although that's why it caught my attention), but I do want to make sure it is intentional and correct.

Reply:
Yeah it's a little bug I found... yes slightly out of scope.